### PR TITLE
[DEV APPROVED] TP: 7031, Comment: Updates Yeast for PR1407 (frontend)

### PR DESCRIPTION
**Pull request by davidtrussler**

### CODE REVIEW

**Comment by slaywell** [comment]
This looks good. Could you just tidy up the commit history, and maybe add a note on the PR to explain what this is doing?

**Comment by slaywell** [comment]
:+1:

File: assets/lib/_variables.scss [hunk 1]
```diff
@@ -185,6 +185,15 @@ $color-email-hover: #e8ae28 !default;
 $color-youtube: #cd201f !default;
 $color-youtube-hover: #B61B1B !default;
 
+// Focus
+$color-focus-BG: $color-yellow-dark !default;
+$color-focus-FG: $color-true-black !default;
```
Comment (slaywell) [inline assets/lib/_variables.scss]: Could we make these all lower-case? That would be more consistent with other variables.

**Comment by slaywell** [comment]
:+1:

**Comment by johnplayer1982** [comment]
:+1:

**Pull request merged**